jobserv/git_poller.py

```diff
@@ -308,18 +334,24 @@ def _trigger(entry: PollerEntry, trigger_name: str,
     }
     p = urlparse(change_params['GIT_URL'])
     url = p.scheme + '://' + p.netloc
+    commit_skipped = False
     if url == 'https://github.com':
-        data['reason'] += '\n' + _github_log(entry.trigger, change_params)
+        commit_summary, commit_skipped = _github_log(entry.trigger, change_params)
+        data['reason'] += '\n' + commit_summary
     elif url in GITLAB_SERVERS:
-        data['reason'] += '\n' + _gitlab_log(entry.trigger, change_params)
+        commit_summary, commit_skipped = _gitlab_log(entry.trigger, change_params)
+        data['reason'] += '\n' + commit_summary
     else:
         capable = _cgit_repos.get(url, True)
         if capable:
-            reason = _cgit_log(entry.trigger, change_params)
-            if reason:
-                data['reason'] += '\n' + reason
+            commit_summary, commit_skipped = _cgit_log(entry.trigger, change_params)
+            if commit_summary:
+                data['reason'] += '\n' + commit_summary
             else:
                 _cgit_repos[url] = False
+    if commit_skipped:
+        log.info("skipped build")
```

Comment:
Might want to make this a little more informative like: `log.info('Skippping build for %s because of skip-ci message', entry.trigger.project)`.

Reply:
Agreed, I've applied these changes.